extensions/adobe/experience/adcloud/conversiondetails.schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/adcloud/conversiondetails",
+  "title": "Conversion Detail",
+  "type": "object",
+  "description": "Adobe AdCloud Ad Conversion Details",
```

Comment:
do we really want a reference to the Adobe AdCloud here?

Reply:
Keeping as is given it's in extension.

Reply:
👍